### PR TITLE
Update Visualize.md

### PR DESCRIPTION
**Pull request by fliklab**
**변경 내용** : '학습 히스토리 시각화' 문단의 단어 및 내용 수정

**내용 수정**
이해하기 쉬운 문장으로 수정(준영님의 sequential.md PR 참고)

**단어 수정**
학습 측정 항목값(training metric value) → 학습 지표값(기존 논의내용에 따름)
학습 정확성 값(training accuracy values) → 학습 정확도
검증 정확성 값(validation accuracy values)→ 검증 정확도
플롯→그래프

**그대로 유지**
검증 손실값(validation loss values)
학습 손실값(training loss values)

**기타**
plot_model는 → plot_model은

### CODE REVIEW

**Comment by fuzzythecat** [comment]
감사합니다!
